src/components/camera-tool.js

```diff
@@ -20,7 +24,29 @@ const pathsMap = {
   }
 };
 
+const isMobileVR = AFRAME.utils.device.isMobileVR();
+
+const VIEWPORT_FPS = 6;
+const VIDEO_FPS = 25;
+// Prefer h264 if available due to faster decoding speec on most platforms
+const videoCodec = ["h264", "vp9", "vp8"].find(
+  codec => window.MediaRecorder && MediaRecorder.isTypeSupported(`video/webm; codecs=${codec}`)
+);
+const videoMimeType = videoCodec ? `video/webm; codecs=${videoCodec}` : null;
+const allowVideo = !!videoMimeType;
```

Comment:
since blitFramebuffer is not available on webgl1 we should also not enable video recording on webgl1 (or add a method of blitting that does work)